source/core/ut_suite_cache.sql

```diff
@@ -45,7 +46,8 @@ create table ut_suite_cache (
   nested table after_each_list store as ut_suite_cache_after_each
   nested table before_test_list store as ut_suite_cache_before_test
   nested table after_test_list store as ut_suite_cache_after_test
-  nested table expected_error_codes store as ut_suite_cache_trhows
+  nested table expected_error_codes store as ut_suite_cache_throws
+  nested table tags store as ut_suite_cache_tags return as locator
```

Comment:
Why `as locator`? Any notable performance gains?
If so, maybe we should consider the same for the remaining nested tables.

Reply:
So I think depends on use in query we use it makes sense as per [docs](https://docs.oracle.com/en/database/oracle/oracle-database/18/adobj/design-considerations-for-collections.html#GUID-1CCB041E-AE30-4732-B39B-695E3291DC57) since we access for every row we look for a child record with a tag.
`For large child sets, the parent row and a locator to the child set can be returned so that the child rows can be accessed on demand; the child sets also can be filtered. Using nested table locators enables you to avoid unnecessarily transporting child rows for every parent. `

Having run locally tests with a just console reporter for locator and non locator run times been comparable , I will try do some autotrace when got some time.